src/PSRule.Rules.Azure/rules/Azure.Storage.Rule.ps1

```diff
@@ -95,6 +95,13 @@ Rule 'Azure.Storage.DefenderCloud.MalwareScan' -Ref 'AZR-000384' -Type 'Microsof
     $Assert.Count($malwareDisabled, '.', 0).Reason($LocalizedData.ResStorageMalwareScanning, $PSRule.TargetName)
 }
 
+# Synopsis: Enable Microsoft Defender for Storage for storage accounts.
+Rule 'Azure.Storage.DefenderCloud' -Ref 'AZR-000386' -Type 'Microsoft.Storage/storageAccounts' -If { $Configuration.AZURE_STORAGE_DEFENDER_PER_ACCOUNT } -Tag @{ release = 'GA'; ruleSet = '2023_06'; 'Azure.WAF/pillar' = 'Security'; } -Labels @{ 'Azure.MCSB.v1/control' = 'DP-2', 'LT-1' } {
```

Comment:
Try the following but we should set this configuration in `Config.Rule.yaml`. Fairly sure that `-Configure` only applies in the rule body, although we could make an update to PSRule to improve that.

```suggestion
Rule 'Azure.Storage.DefenderCloud' -Ref 'AZR-000386' -Type 'Microsoft.Storage/storageAccounts' -If { $Configuration.AZURE_STORAGE_DEFENDER_PER_ACCOUNT -eq $True } -Tag @{ release = 'GA'; ruleSet = '2023_06'; 'Azure.WAF/pillar' = 'Security'; } -Labels @{ 'Azure.MCSB.v1/control' = 'DP-2', 'LT-1' } {
```

Reply:
@BernieWhite What you suggested did work, but I have no understanding of why since I had it in the configuration file, so why accessing the property as is does not emit boolean? A script block returns the output of the statements or expressions inside, so why is it only emitting with the equality operator?

https://github.com/Azure/PSRule.Rules.Azure/blob/61b3657832c4a883a178def1b3d4417c3192123d/tests/PSRule.Rules.Azure.Tests/ps-rule-options.yaml#L106